index.bs

```diff
@@ -2462,6 +2663,59 @@ getter must return true if this [=/object store handle=]'s
 and false otherwise.
 
 
+<div class=note>
+  The following methods throw {{ReadOnlyError}} if called within a
+  [=read-only transaction=], and {{TransactionInactiveError}} if
+  called when the [=/transaction=] is not [=transaction/active=].
+
+  <dl class=domintro>
+    <dt>
+      <var>request</var> = <var>store</var>
+          . {{IDBObjectStore/put()|put}}(<var>value</var> [, <var>key</var>])
+    </dt>
+    <dt>
+      <var>request</var> = <var>store</var>
+          . {{IDBObjectStore/add()|add}}(<var>value</var> [, <var>key</var>])
+    </dt>
+    <dd>
+       Adds or updates a [=record=] in |store| with the given |value|
+       and |key|. If the store uses [=in-line keys=] then |key| must
+       not be specified, or a {{DataError}} will be thrown.
+
+       If {{IDBObjectStore/put()}} is used, any existing [=record=]
+       with the [=/key=] will be replaced. If {{IDBObjectStore/add()}}
+       is used, and if a [=record=] with the [=/key=] already exists
+       the |request| will fail, with |request|'s {{IDBRequest/error}}
+       set to {{ConstraintError}}.
+
+       If successful, |request|'s {{IDBRequest/result}} will be the
+       [=record=]'s [=/key=].
+    </dd>
+
+    <dt>
+      <var>request</var> = <var>store</var> .
+          {{IDBObjectStore/delete()|delete}}(<var>query</var>)
+    </dt>
+    <dd>
+       Deletes [=records=] in |store| with the given
+       [=/key=] or in the given [=key range=] in |query|.
+
+       If successful, |request|'s {{IDBRequest/result}} will
+       be undefined.
```

Comment:
I wonder if, in developer-facing documentation at least, `undefined` should be in code font? I'm not sure, since we have a pretty strong precedent of leaving it in normal font inside algorithms.